### ISSUE TITLE
I cant install ps module

### ISSUE THREAD

**Issue by akolomyts** [R3MRUM/PSDecode#14]
I followed the instructions:

> Created a directory
> Copied your .psm1
> restarted the PS shell

`PS C:\> PSDecode -dump -beautify -verbose .\vir.ps1
Имя "PSDecode" не распознано как имя командлета, функции, файла скрипта или выполняемой программы. Проверьте правильность написания имени, а также наличие и правильность пути, после чего повторите попытку.
строка:1 знак:9
+ PSDecode <<<<  -dump -beautify -verbose .\vir.ps1
    + CategoryInfo          : ObjectNotFound: (PSDecode:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException`

VirtualBox 6.1.18, Windows 7 x32, SP1

You can help me to install your ps module?
Thanks

**Comment by akolomyts** [comment]
I use command to change profile:

1. `Get-ExecutionPolicy `
and get answer - **Restricted**
2. To use another modules need policy - RemoteSigned 
`Set-ExecutionPolicy RemoteSigned `
3. and then need do:
`Import-module PSDecode`
4. And get error (win 7 x32)
```
PS C:\Users\Admin> Import-Module PSDecode
Необходимо предоставить выражение для значения справа от оператора "-".
C:\Windows\system32\WindowsPowerShell\v1.0\Modules\PSDecode\PSDecode.psm1:279 знак:33
+             if($Command[$char] - <<<< in $str_quotes -and -not $in_str){
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : ExpectedValueExpression

Import-Module : Указанный модуль "PSDecode" не был загружен, так как ни в одном из каталогов модулей не был обнаружен действительный файл модуля.
строка:1 знак:14
+ Import-Module <<<<  PSDecode
    + CategoryInfo          : ResourceUnavailable: (PSDecode:String) [Import-Module], FileNotFoundException
    + FullyQualifiedErrorId : Modules_ModuleNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand
```

PS on Win10 x64 20H2 work good.

**Comment by nokae** [comment]
This is old, but might help other people if answered. I don't think RemoteSigned is going to work to use this module because that execution policy "requires a digital signature from a trusted publisher." Try Bypass instead, it worked for me. [Here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.2) is an explanation of each execution policy.

**Comment by huornlmj** [comment]
Yes, @nokae is correct.  On Windows 10, this works.  Run PowerShell as administrator and then...
`Set-ExecutionPolicy Bypass`
Answer Yes
Then...
`Import-module PSDecode`
